tests/publiccloud/prepare_instance.pm

```diff
@@ -54,6 +54,11 @@ sub run {
     if (is_sle('<15') && is_azure) {
         $instance->ssh_assert_script_run(q(sudo sed -i "/Defaults targetpw/s/^#//" /etc/sudoers));
     }
+
+    # For maintenance test runs, the testapi user (typically 'bernhard') is expected to be allowed to run sudo commands
+    if (check_var('PUBLIC_CLOUD_QAM', '1')) {
+        $instance->ssh_assert_script_run("sudo echo \"$testapi::username ALL=(ALL) ALL\" > /etc/sudoers.d/010_openqa");
```

Comment:
We can drop sudo here:

```suggestion
        $instance->ssh_assert_script_run("echo \"$testapi::username ALL=(ALL) ALL\" > /etc/sudoers.d/010_openqa");
```

Reply:
Nope we cannot because those commands are run as e.g. `azureuser`, who doesn't have write permissions to `/etc/sudoers.d`.